### ISSUE TITLE
It's need to add tqdm console switch (`disable=True`)

### ISSUE THREAD

**Issue by ucomru** [AlJohri/docx2pdf#89]
There are a lot of problems with console logging tqdm.

I think it's need to add param `log: bool = True` to the `docx2pdf.convert` function

<img width="452" alt="1" src="https://github.com/AlJohri/docx2pdf/assets/101626426/da25c0b1-8e46-48c2-9e43-853321fe1f49">

And then add `disable=log` to the `tqdm` method:

```python
...
pbar = tqdm(total=total, disable=log)
...
```

**Comment by ShekharNarayanan** [comment]
Yes please! Converting to pdf in loops for a large number of files fills the terminal with unnecessary logs.
